src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <!-- <h1>Search bar airbnb</h1> -->
+    <div class="container big-container">
+      <form
+        class="forma big-forma"
```

Comment:
use only english words

```suggestion
        class="form big-form"
```